### PR TITLE
Feature/fix datapacket endpoints

### LINKED ISSUE #423: Fix datapacket endpoints
That's not how http methods are supposed to be used =D

```
webapp.get '/datapacket/exists/:id', urlParser, dataPackets.exists
webapp.get '/datapacket/get/:id', urlParser, dataPackets.get
webapp.put '/datapacket/put/:id', jsonParser, dataPackets.put
webapp.get '/datapacket/create', urlParser, dataPackets.create
webapp.delete '/datapacket/delete', urlParser, dataPackets.delete
```

This should be:

```
webapp.route('/datapacket/:id')
  .get dataPackets.get
  .put dataPackets.put
  .delete dataPackets.delete

webapp.post '/datapacket', dataPackets.create
```

The action shouldn't be encoded in the request method and the endpoint.

### PR DESCRIPTION
**Pull request by Steditor**
Closes #423

### CODE REVIEW

File: src/server/main.coffee [hunk 1]
```diff
@@ -177,11 +177,11 @@ module.exports.setupRouting = ->
 	webapp.get '/model/get/:hash', urlParser, modelStorageApi.getModel
 	webapp.post '/model/submit/:hash', rawParser, modelStorageApi.saveModel
 
-	webapp.get '/datapacket/exists/:id', urlParser, dataPackets.exists
-	webapp.get '/datapacket/get/:id', urlParser, dataPackets.get
-	webapp.put '/datapacket/put/:id', jsonParser, dataPackets.put
 	webapp.get '/datapacket/create', urlParser, dataPackets.create
```
Comment (adius) [inline src/server/main.coffee]: http://stackoverflow.com/questions/29307616/rest-api-endpoint-for-retrieving-empty-object

And the official RFC 7231:
> Creating a new resource that has yet to be identified by the origin server

I'd vote for `webapp.post /datapacket/new`

**Comment by Steditor** [comment]
@adius everything done - pls revisit!

**Comment by adius** [comment]
```txt
Chrome 43.0.2341 (Mac OS X 10.10.3) dataPacket client cache dataPacket creation should call the server to create a dataPacket FAILED
	Error: the object {
	  "responseText": "NOT FOUND"
	  "status": 404
	  "statusText": "Not Found"
	} was thrown, throw an Error :)
	    at lib$es6$promise$$internal$$tryCatch (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5925:16)
	    at lib$es6$promise$$internal$$invokeCallback (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5937:17)
	    at lib$es6$promise$$internal$$publish (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5908:11)
	    at lib$es6$promise$$internal$$publishRejection (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5858:7)
	    at MutationObserver.lib$es6$promise$asap$$flush (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5719:9)
Chrome 43.0.2341 (Mac OS X 10.10.3): Executed 1 of 19 (1 FAILED) (0 secs / 0.011 secs)
Chrome 43.0.2341 (Mac OS X 10.10.3) dataPacket client cache dataPacket creation should cache a newly created dataPacket FAILED
	Error: the object {
	  "responseText": "NOT FOUND"
	  "status": 404
	  "statusText": "Not Found"
	} was thrown, throw an Error :)
	    at lib$es6$promise$$internal$$tryCatch (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5925:16)
	    at lib$es6$promise$$internal$$invokeCallback (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5937:17)
	    at lib$es6$promise$$internal$$publish (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5908:11)
	    at lib$es6$promise$$internal$$publishRejection (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5858:7)
	    at MutationObserver.lib$es6$promise$asap$$flush (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5719:9)
Chrome 43.0.2341 (Mac OS X 10.10.3): Executed 2 of 19 (2 FAILED) (0 secs / 0.038 secs)
Chrome 43.0.2341 (Mac OS X 10.10.3) dataPacket client cache dataPacket creation should fail if the server cannot create a dataPacket FAILED
	AssertionError: expected { Object (status, statusText, ...) } to have a property 'status' of 500, but got 404
	    at absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:18104:40
	    at lib$es6$promise$$internal$$tryCatch (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5925:16)
	    at lib$es6$promise$$internal$$invokeCallback (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5937:17)
	    at lib$es6$promise$$internal$$publish (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5908:11)
	    at lib$es6$promise$$internal$$publishRejection (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5858:7)
	    at MutationObserver.lib$es6$promise$asap$$flush (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5719:9)
Chrome 43.0.2341 (Mac OS X 10.10.3): Executed 5 of 19 (3 FAILED) (0 secs / 1.072 secs)
Chrome 43.0.2341 (Mac OS X 10.10.3) dataPacket client cache dataPacket existence checks should answer from cache if dataPacket is known FAILED
	Error: the object {
	  "responseText": "NOT FOUND"
	  "status": 404
	  "statusText": "Not Found"
	} was thrown, throw an Error :)
	    at lib$es6$promise$$internal$$tryCatch (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5925:16)
	    at lib$es6$promise$$internal$$invokeCallback (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5937:17)
	    at lib$es6$promise$$internal$$publish (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5908:11)
	    at lib$es6$promise$$internal$$publishRejection (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5858:7)
	    at MutationObserver.lib$es6$promise$asap$$flush (absolute/var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/1eeaffdc9dad90fd196035f859e22cc477e180f5.browserify?8cef6b650ae808c8b6a0a9dd187c905c7e7a2907:5719:9)
Chrome 43.0.2341 (Mac OS X 10.10.3): Executed 19 of 19 (4 FAILED) (8.282 secs / 8.168 secs)
```

**Comment by Steditor** [comment]
@adius done, third try :)

**Pull request merged**